### ISSUE TITLE
Move valuable sample code to relevant packages or framework

### ISSUE THREAD

**Issue by guidezpl** [flutter/gallery#1077]

**Comment by stuartmorgan** [comment]
> * Motion → `animations` cc @stuartmorgan

@goderbauer is the primary owner of `animations`.

**Comment by goderbauer** [comment]
@guidezpl Could you link to the motion sample code that you have in mind?

**Comment by guidezpl** [comment]
They're in https://github.com/flutter/gallery/tree/main/lib/demos/reference, each starting with `motion_`

**Comment by goderbauer** [comment]
Those examples already exist in package:animations, see https://github.com/flutter/packages/tree/main/packages/animations/example/lib. I suspect, that they were actually copied from that package (with minor modifications like adding localizations) to the gallery 😄. I think we can remove these from the list of potential candidate for a move.

**Comment by guidezpl** [comment]
Indeed, with slightly modified names and a combined axis demo. For posterity, the demo looked as follows

![image](https://github.com/flutter/gallery/assets/6655696/82951795-38ef-40a0-a73c-cc65cac03799)
